packaging/ubuntu-16.04/snapd.maintscript

```diff
@@ -2,3 +2,4 @@
 # we used to ship a custom grub config that is no longer needed
 rm_conffile /etc/grub.d/09_snappy 1.7.3ubuntu1
 rm_conffile /etc/ld.so.conf.d/snappy.conf 2.0.7~
+rm_conffile /etc/apparmor.d/usr.lib.snapd.snap-confine 2.23.6~
```

Comment:
seems rm_conffile doc about this changed over time (the mysteries of life), are we really releasing 2.23.6 ? I lost count it seems